src/Symfony/Component/Translation/Formatter/DefaultMessageFormatter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Formatter;
+
+use Symfony\Component\Translation\MessageSelector;
+
+/**
+ * @author Guilherme Blanco <guilhermeblanco@hotmail.com>
+ */
+class DefaultMessageFormatter implements MessageFormatterInterface
```

Comment:
I don't like the name ``Default`` here. The name should reflect what makes this implementation specific, not that it is used by default (we could decide to change the default in future versions)